### PR TITLE
KBI0004: FAQ: is it normal that files are moved to .git/annex/objects when converting existing data into a DataLad dataset?

### PR DESCRIPTION
**Pull request by christian-monch**
This is an excerpt from @jsheunis' answer on Neurostars (https://neurostars.org/t/using-datalad-for-managing-data-on-cluster/24494/2).

I think a lot of new datalad users will ask themselves the same question and it should be an FAQ-entry

### CODE REVIEW

**Review by adswa — APPROVED** [review]
cool, I agree that its useful!

File: kbi/0004/index.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+This knowledge-base item explains why files are moved into ``.git/annex/objects``, when
+a DataLad dataset is created from an existing dataset, or when a dataset with new files
+is saved.
+
+
+Question:
+---------
+
+When I convert an existing folder to a DataLad dataset, all files are moved to ``./.git/annex/objects/`` and
```
Comment (mih) [inline kbi/0004/index.rst:19]:
```suggestion
On my Mac or Linux machine, when I convert an existing folder to a DataLad dataset, all files are moved to ``./.git/annex/objects/`` and
```

Maybe hint at this being a common, but still not universal case?

File: kbi/0004/index.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+the data content is performed by ``git-annex``.
+
+This setup creates a modular and portable dataset (the ``git`` repository) which contains information about the versions
+and history of data inside the dataset, while the actual data content is managed by ``git-annex``. File content that is
+placed under management of ``git-annex`` will be moved into ``./.git/annex/objects/`` and a symbolic link (symlink) to
+the content will remain in the original path. This symbolic link will be tracked by ``git``.
+
+The separation of version management (done by ``git``) and content management (done by ``git-annex``)) make a DataLad
+dataset very flexibld. You can for example share the dataset (``git`` repository) publicly, while keeping the contents
+safe on your cluster. People can then access the dataset (with ``datalad clone``) and download individual files in the
```
Comment (mih) [inline kbi/0004/index.rst:39]: It is probably more accessible to not assume readers know what a "cluster" is in this context, or why one would have one to begin with.

File: kbi/0004/index.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+dataset (with ``datalad get``) if they have access credentials for that location on your cluster.
```
Comment (jsheunis) [inline kbi/0004/index.rst:40]:
```suggestion
dataset (with ``datalad get``) if they have access credentials for that particular storage location.
```

File: kbi/0004/index.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+However, you can use configurations to specify how datalad should commit/manage your data. You might want to commit all
```
Comment (jsheunis) [inline kbi/0004/index.rst:42]:
```suggestion
However, you can use configurations to specify how DataLad should commit/manage your data. You might want to commit all
```

File: kbi/0004/index.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+ensures that ``git`` only needs to track the links, which are quite small in size. The task of transporting and handling
+the data content is performed by ``git-annex``.
+
+This setup creates a modular and portable dataset (the ``git`` repository) which contains information about the versions
+and history of data inside the dataset, while the actual data content is managed by ``git-annex``. File content that is
+placed under management of ``git-annex`` will be moved into ``./.git/annex/objects/`` and a symbolic link (symlink) to
+the content will remain in the original path. This symbolic link will be tracked by ``git``.
+
+The separation of version management (done by ``git``) and content management (done by ``git-annex``)) make a DataLad
+dataset very flexibld. You can for example share the dataset (``git`` repository) publicly, while keeping the contents
```
Comment (mslw) [inline kbi/0004/index.rst:38]:
```suggestion
dataset very flexible. You can for example share the dataset (``git`` repository) publicly, while keeping the contents
```

File: kbi/0004/index.rst [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+safe elsewhere. People can then access the dataset (with ``datalad clone``) and download individual files in the
+dataset (with ``datalad get``) if they have access credentials for that particular storage location.
+
+However, you can use configurations to specify how DataLad should commit/manage your data. You might want to commit all
+your files to ``git`` (unless they are large, too numerous, or unless you don't want to make them available to
+everybody who clones the ``git`` repository). You could also let every file be managed by ``git-annex``. Usually only
+large binary files will be managed by ``git-annex`` by default.
```
Comment (mslw) [inline kbi/0004/index.rst:45]: What kind of default are we talking about in the last sentence? DataLad's default would be to manage all files with git-annex.

**Review by mslw — COMMENTED** [review]
It's a nice addition to the knowledge base. I left two minor comments.

File: kbi/0004/index.rst (hunk 6 above)
Comment (mih) [inline kbi/0004/index.rst:45]:
```suggestion
everybody who clones the ``git`` repository). You could also let every file be managed by ``git-annex``.
```

Comment (mih) [reply kbi/0004/index.rst:45]: Indeed. But given that this is not strictly limited to DataLad, I am simply proposing a removal of that last sentence below.

**Review by mih — APPROVED** [review]
I think this is ready to go.

**Pull request merged**